setup.py

```diff
@@ -58,16 +54,20 @@ def build_extensions(self):
         comp = self.compiler.compiler_type
         if comp in ('unix', 'cygwin', 'mingw32'):
             # Check if build is with OpenMP
-            if use_omp:
-                extra_compile_args = ['-std=c99', '-O3', '-fopenmp']
-                extra_link_args=['-lgomp']
+            if USE_OMP and IS_OSX:
+                if IS_OSX:
```

Comment:
I could be missing something, but this second `if IS_OSX` isn't needed, right? We already know it is True since `if USE_OMP and IS_OSX`.

Reply:
Or should this actually be `IS_CLANG` if we could figure out how to get that information?